tests/test-cases.js

```diff
@@ -2716,7 +2725,7 @@ describe('Custom Tests', function() {
 		const blob = new Blob([
 			`
 				importScripts('${papaParseScriptPath}');
-				
+
```

Comment:
This seems unreleated to the change. Could you please avoid modifyinng unrelated lines? Thanks